examples/complete/main.tf

```diff
@@ -104,3 +104,27 @@ module "postgresql_db" {
     }
   ]
 }
+
+##############################################################################
+# VPE
+##############################################################################
+
+resource "ibm_is_security_group" "sg1" {
+  name = "${var.prefix}-sg1"
+  vpc  = ibm_is_vpc.example_vpc.id
+}
+
+resource "ibm_is_virtual_endpoint_gateway" "pgvpe" {
+  name = "vpe-to-access-pg"
```

Comment:
Are you only allowed one of these per account (or are duplicate names permitted)?

For validation involving the pipelines running in parallel and to avoid accidents (eg in nuke) being impacted by duplicate names, include var.prefix?